fabtests/pytest/efa/test_rdm.py

```diff
@@ -1,6 +1,5 @@
 import pytest
 
-from default.test_rdm import test_rdm
```

Comment:
where is `test_rdm` used in this script? Just import it can make it run?

Reply:
Yes, simple import it makes it to run.

Reply:
The implementation of the test is in `pytest/default/test_rdm.py`

Reply:
Is it possible to add a marker for this function to skip it?

Reply:
I think added a skip will  confuse the normal users.

Reply:
I see. Thanks.